server/routes/appointments/appointmentsController.ts

```diff
@@ -183,32 +184,41 @@ export default class AppointmentsController {
     if (draft.data === null) {
       throw new Error('Draft data was unexpectedly null when submitting appointment')
     }
-
-    try {
-      await this.interventionsService.scheduleSupplierAssessmentAppointment(
-        res.locals.user.token.accessToken,
-        supplierAssessment.id,
-        draft.data
+    const appointmentScheduleDetails = this.extractAppointmentSchedulingDetails(fetchResult.draft)
```

Comment:
yeah. This was how it was done in delivery session so I just kept it the same. All that method was doing is returning the scheduling details or null. I guess it could just be done the other way ad keep it all together.